src/auspex/filters/io.py

```diff
@@ -19,7 +19,10 @@
 import re
 import pandas as pd
 from shutil import copyfile
-from ruamel.yaml import YAML
+try:
+    from ruamel.yaml import YAML
+except:
+    from ruamel_yaml import yaml_object
```

Comment:
should this be `as YAML` or something?

Reply:
In `config.py` we have, for example:
```python
try:
    import ruamel.yaml as yaml
except:
    import ruamel_yaml as yaml
```

Reply:
I think you're right. I got confused with all these ~namesakes

Reply:
actually, I don't think `YAML` is used either...